deploy/charts/secret-manager/values.yaml

```diff
@@ -12,6 +12,36 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+# Optional ApiServer host address to connect to. If not specified, autoconfiguration will be attempted
+apiServerHost: ""
+# Path to a kubeconfig. Only required if out-of-cluster
+kubeConfig: ""
+# If set, this limits the scope of secret-manager to a single namespace and
+# ClusterSecretStores are disabled. If not specified, all namespaces will be watched
+namespace: ""
+# Leader Election configuration
+leaderConfig:
+  # If true, secret-manager will perform leader election between instances to ensure no more
+  # than one instance of secret-manager operates at a time
+  elect: true
+  # Namespace used to perform leader election. Only used if leader election is enabled
+  namespace: "kube-system"
```

Comment:
This should probably be an empty string and default to `.Release.Namespace`. I think the flag defaults to kube-system just so there aren't errors if the namespace doesnt exist.